sdk/communication/azure-communication-administration/pom.xml

```diff
@@ -59,6 +59,11 @@
       <artifactId>azure-core</artifactId>
       <version>1.11.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->    
+    </dependency>
```

Comment:
Client libraries should not have a dependency on the azure-identity library. Is this needed by the library or should it be a test dependency?

Reply:
It should be a test dependency, you're right

Reply:
@schaabs Done!